contracts/CurveGlobal.sol

```diff
@@ -0,0 +1,77 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity 0.6.12;
+pragma experimental ABIEncoderV2;
+
+import "./StrategyConvexFactoryClonable.sol";
+
+interface Registry{
+    function newExperimentalVault(address token, address governance, address guardian, address rewards, string memory name, string memory symbol) external returns (address);
+}
+
+interface Vault{
+    
+    function setGovernance(address) external;
+    function addStrategy(address, uint, uint, uint, uint) external;
+}
+
+contract CurveGlobal{
+
+    address owner = 0xFEB4acf3df3cDEA7399794D0869ef76A6EfAff52;
+    Registry public registry = Registry(address(0x50c1a2eA0a861A967D9d0FFE2AE4012c2E053804));
+    IConvexDeposit public convexDeposit = IConvexDeposit(0xF403C135812408BFbE8713b5A23a04b3D48AAE31);
+    address public sms = address(0x16388463d60FFE0661Cf7F1f31a7D658aC790ff7);
+    address public ychad = address(0xFEB4acf3df3cDEA7399794D0869ef76A6EfAff52);
+    address public devms = address(0x846e211e8ba920B353FB717631C015cf04061Cc9);
+    address public treasury = address(0x93A62dA5a14C80f265DAbC077fCEE437B1a0Efde);
```

Comment:
why don't you pass value in the clone method instead of hardcoding in the contract?